### PR TITLE
add tidb in chehaoduo

### PR DESCRIPTION
**Pull request by ran-huang**
Signed-off-by: Ran <huangran@pingcap.com>

### CODE REVIEW

File: top-car-trading-platform-chooses-scale-out-database-as-mysql-alternative.md [hunk 1]
```diff
@@ -0,0 +1,307 @@
+---
+title: Top Car Trading Platform Chooses a Scale-out Database as MySQL Alternative
```
Comment (CaitinChen) [inline top-car-trading-platform-chooses-scale-out-database-as-mysql-alternative.md:2]:
```suggestion
title: A Top Car Trading Platform Chooses a Scale-out Database as a MySQL Alternative
```

File: top-car-trading-platform-chooses-scale-out-database-as-mysql-alternative.md [hunk 2]
```diff
@@ -0,0 +1,307 @@
...
+author: ["Chehaoduo"]
+date: 2021-01-12
+summary: Chehaoduo, the largest auto trading platform in China, migrated from MySQL to TiDB and got rid of the troublesome process of MySQL sharding and schema changes.
+image: /images/blog/chehaoduo-mysql-alternative-scale-out-database.png
+url: /success-stories/top-car-trading-platform-chooses-scale-out-database-as-mysql-alternative/
```
Comment (CaitinChen) [inline top-car-trading-platform-chooses-scale-out-database-as-mysql-alternative.md:7]: We can remove this line now.

File: top-car-trading-platform-chooses-scale-out-database-as-mysql-alternative.md [hunk 3]
```diff
@@ -0,0 +1,307 @@
...
+customer: Chehaoduo
+customerCategory: Internet
+logo: /images/blog/customers/chehaoduo-logo.png
+---
+
+**Industry:** Automobile
+
+**Author:** Kevin Ge, Jiansheng Han, and Sheyang Wang (TiDB virtual team at Chehaoduo)
```
Comment (CaitinChen) [inline top-car-trading-platform-chooses-scale-out-database-as-mysql-alternative.md:15]:
```suggestion
**Authors:** Kevin Ge, Jiansheng Han, and Sheyang Wang (TiDB virtual team at Chehaoduo)
```

File: top-car-trading-platform-chooses-scale-out-database-as-mysql-alternative.md [hunk 4]
```diff
@@ -0,0 +1,307 @@
...
+
+![Chehaoduo banner](media/chehaoduo-mysql-alternative-scale-out-database.png)
```
Comment (CaitinChen) [inline top-car-trading-platform-chooses-scale-out-database-as-mysql-alternative.md:17]:
```suggestion
![Chehaoduo uses a scale-out MySQL alternative database](media/chehaoduo-mysql-alternative-scale-out-database.png)
```

Comment (CaitinChen) [reply top-car-trading-platform-chooses-scale-out-database-as-mysql-alternative.md:17]: For SEO

File: top-car-trading-platform-chooses-scale-out-database-as-mysql-alternative.md [hunk 5]
```diff
@@ -0,0 +1,307 @@
...
+
+[Chehaoduo](https://www.crunchbase.com/organization/guazi-com) is an online trading platform for both new cars and personal used cars. Founded in 2015, it is now the largest auto trading platform in China, valued at $9 billion in its series D round of funding last year.
+
+In the early stages of Chehaoduo, to quickly adapt to our application development, we chose MySQL as our major database. However, as our business evolved, we were greatly troubled by the complication of MySQL sharding and schema changes. In face of this dilemma, we found an alternative database to MySQL: [TiDB](https://docs.pingcap.com/tidb/stable), an open-source, MySQL compatible database that scales out to hold massive data.
+
+In this post, I’ll share with you why we chose TiDB and how it empowers our application to provide better service for our customers.
```
Comment (CaitinChen) [inline top-car-trading-platform-chooses-scale-out-database-as-mysql-alternative.md:23]: Please update all quotes to straight quotes.

File: top-car-trading-platform-chooses-scale-out-database-as-mysql-alternative.md (hunk 2 above)
Comment (ran-huang) [inline top-car-trading-platform-chooses-scale-out-database-as-mysql-alternative.md:7]:
```suggestion
```

Comment (ran-huang) [reply top-car-trading-platform-chooses-scale-out-database-as-mysql-alternative.md]: done

File: top-car-trading-platform-chooses-scale-out-database-as-mysql-alternative.md (hunk 5 above)
Comment (ran-huang) [reply top-car-trading-platform-chooses-scale-out-database-as-mysql-alternative.md]: fixed

File: top-car-trading-platform-chooses-scale-out-database-as-mysql-alternative.md [hunk 6]
```diff
@@ -0,0 +1,307 @@
+---
+title: A Top Car Trading Platform Chooses a Scale-out Database as a MySQL Alternative
+author: ["Chehaoduo"]
+date: 2021-01-12
+summary: Chehaoduo, the largest auto trading platform in China, migrated from MySQL to TiDB and got rid of the troublesome process of MySQL sharding and schema changes.
+tags: ['Database selection','MySQL sharding','MySQL compatibility','Scalability']
```
Comment (CaitinChen) [inline top-car-trading-platform-chooses-scale-out-database-as-mysql-alternative.md:6]:
```suggestion
tags: ['Database selection', 'MySQL sharding', 'MySQL compatibility', 'Scalability']
```

**Review by CaitinChen — APPROVED** [review]
LGTM

**Pull request merged**